Frontend/ui-driver/src/Screens/UploadDrivingLicenseScreen/ComponentConfig.purs

```diff
@@ -51,7 +51,7 @@ primaryButtonConfig state = let
       }
       , width = MATCH_PARENT
       , background = Color.black900
-      , margin = Margin 15 0 15 30
+      , margin = if (state.props.openHowToUploadManual && (not state.data.cityConfig.uploadRCandDL)) then Margin 15 0 15 10 else Margin 15 0 15 30
```

Comment:
put `state.props.openHowToUploadManual && not state.data.cityConfig.uploadRCandDL` in a variable and remove brackets then use the variable